sql/sql_class.h

```diff
@@ -1510,6 +1510,24 @@ class THD : public MDL_context_owner,
   NET net;        // client connection descriptor
   String packet;  // dynamic buffer for network I/O
  public:
+  bool has_net_vio() const noexcept { return net.vio != nullptr; }
+  Vio *get_net_vio() const noexcept { return net.vio; }
+  bool has_net_vio_ssl_arg() const noexcept {
+#ifdef HAVE_OPENSSL
```

Comment:
IMHO in 8.0 HAVE_OPENSSL should be assumed to be always defined

Reply:
Actually, this comes from `include/violite.h`
```c++
#ifdef HAVE_OPENSSL
  void *ssl_arg = {nullptr};
#endif
```
Not sure if these are just refactoring leftovers, but I just made it match.

Reply:
On the other hand here is a line from top-level `CMakeLists.txt`
```
ADD_DEFINITIONS(-DHAVE_OPENSSL)            # TODO: remove #ifdef from C++ code
```